pom.xml

```diff
@@ -31,6 +31,11 @@
       <name>Manoj Thakkar</name>
       <email>mthakkar@cloudera.com</email>
     </developer>
+    <developer>
+      <id>fsta</id>
+      <name>Frano Stanisic</name>
+      <email>frano.stanisic@serato.com</email>
+    </developer>
```

Comment:
This block is used for maintainers, IIUC you're not a maintainer on this plugin. If you want your name be added, use <contributors> instead.